### PR TITLE
TP-1264-unify-loading-screens

### PR DESCRIPTION
**Pull request by sharlotta93**
Add new loading screen

### CODE REVIEW

File: src/hostedPages/layouts/HostedPageLayout/LoadActivityPlaceholder/loadActivityPlaceholder.module.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+  }
+  
+  .loading_indicator {
+    width: 100%;
+    height: 3.125rem;
+    border-radius: 0.625rem;
```
Comment (nckhell) [inline src/hostedPages/layouts/HostedPageLayout/LoadActivityPlaceholder/loadActivityPlaceholder.module.scss]:
```suggestion
    border-radius: var(--awell-border-radius-xl);
```

File: src/hostedPages/layouts/HostedPageLayout/LoadActivityPlaceholder/loadActivityPlaceholder.module.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+  }
+  
+  .loading_indicator {
+    width: 100%;
+    height: 3.125rem;
+    border-radius: 0.625rem; 
+    margin: 0.4rem 0;
```
Comment (nckhell) [inline src/hostedPages/layouts/HostedPageLayout/LoadActivityPlaceholder/loadActivityPlaceholder.module.scss]:
```suggestion
    margin: var(--awell-spacing-2) 0;
```

File: src/hostedPages/layouts/HostedPageLayout/LoadActivityPlaceholder/loadActivityPlaceholder.module.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    }
+  }
+   
+   /* Styles for smaller screens (mobile) */
+   @media (max-width: 767px) {
+    .loading_indicators {
+       min-width: 336px;
+       padding: 2rem 1rem;
+    }
+   }
```
Comment (nckhell) [inline src/hostedPages/layouts/HostedPageLayout/LoadActivityPlaceholder/loadActivityPlaceholder.module.scss]: I think you could just do this:


```suggestion
.loading_indicators {
  max-width: var(--awell-activity-max-width);
  padding: var(--awell-spacing-8) var(--awell-spacing-4);
}
```

Comment (nckhell) [reply src/hostedPages/layouts/HostedPageLayout/LoadActivityPlaceholder/loadActivityPlaceholder.module.scss]: And JFYI

This
```scss
  @media (min-width: 768px) {
    .loading_indicators {
       max-width: 570px;
       padding: 2rem 0;
    }
  }
   
   /* Styles for smaller screens (mobile) */
   @media (max-width: 767px) {
    .loading_indicators {
       min-width: 336px;
       padding: 2rem 1rem;
    }
   }
```

Can also be written as just this:

```scss
.loading_indicators {
  min-width: 336px;
  padding: 2rem 1rem;

  @media (min-width: 768px) {
     max-width: 570px;
     padding: 2rem 0;
  }
}
```

**Review by nckhell — APPROVED** [review]

File: src/hostedPages/activities/form/ConversationalForm.tsx [hunk 4]
```diff
@@ -8,6 +8,7 @@ import layoutClasses from '../../layouts/HostedPageLayout/hostedPageLayout.modul
 import { FormProps } from '../../../types/form'
 import { useConversationalForm } from '../../../hooks'
 import { Question, UserQuestionType } from '../../../types'
+import { LoadActivityPlaceholder } from '../../layouts/HostedPageLayout/LoadActivityPlaceholder'
```
Comment (danijel) [inline src/hostedPages/activities/form/ConversationalForm.tsx:11]: np: we have 3 imports from same `../../layouts/HostedPageLayout` folder and an index file in it, can we please orgranize import so we import from that folder when we can instead from separate files.

File: src/hostedPages/activities/form/TraditionalForm.tsx [hunk 5]
```diff
@@ -8,6 +8,7 @@ import { FormProps } from '../../../types/form'
 import { UserQuestionType } from '../../../types'
 import { HostedPageFooter } from '../../layouts/HostedPageLayout/HostedPageFooter'
 import { useTheme } from '../../../atoms/themeProvider/ThemeProvider'
+import { LoadActivityPlaceholder } from '../../layouts/HostedPageLayout/LoadActivityPlaceholder'
```
Comment (danijel) [inline src/hostedPages/activities/form/TraditionalForm.tsx:11]: same import nit-pick

**Review by danijel — APPROVED** [review]

File: src/hostedPages/layouts/HostedPageLayout/LoadActivityPlaceholder/loadActivityPlaceholder.module.scss (hunk 3 above)
Comment (sharlotta93) [reply src/hostedPages/layouts/HostedPageLayout/LoadActivityPlaceholder/loadActivityPlaceholder.module.scss]: this also doesn't quite work. If I had more time I would try to condense it but I rather merge it now and move to testing.

**Pull request merged**